bazel/linux/defs.bzl

```diff
@@ -94,6 +94,9 @@ def _kernel_modules(ctx):
     if extra_symbols:
         extra.append("KBUILD_EXTRA_SYMBOLS=\"%s\"" % (extra_symbols))
 
+    extra.append('BAZEL_BIN_DIR="%s"' % ctx.bin_dir.path)
+    extra.append('BAZEL_GEN_DIR="%s"' % ctx.genfiles_dir.path)
```

Comment:
how is BAZEL_GEN_DIR used in the Makefile? Maybe it should just append -I cflags correctly directly, like on line 132? 

Basically: if there's a way to get the Makefile / build do the right thing without adding knowledge of bazel magic in the Makefile, that would be strongly preferred. If not, this patch looks great.

Reply:
The problem is that we need to point to the correct place of files to compile are. It is not a matter of include directories.